.kokoro

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+
+# Display commands to stderr.
+set -x
+
+KOKORO_RUNNER_VERSION="v3.*"
+
+fix_bazel_imports() {
+  if [ -z "$KOKORO_BUILD_NUMBER" ]; then
+    tests_dir_prefix=""
+  else
+    tests_dir_prefix="github/repo/"
+  fi
+
+  # Fixes a bug in bazel where objc_library targets have a _ prefix.
+  find "${tests_dir_prefix}tests/unit" -type f -name '*.swift' -exec sed -i '' -E "s/import Motion(.+)/import _Motion\1/" {} + || true
+  # CocoaPods requires that public headers of dependent pods be implemented using framework import
+  # notation, while bazel requires that dependency headers be imported with quoted notation.
+  find "${tests_dir_prefix}src" -type f -name '*.h' -exec sed -i '' -E "s/import <MotionInterchange\/MotionInterchange\.h>/import \"MotionInterchange.h\"/" {} + || true
+  stashed_dir=$(pwd)
+  reset_imports() {
+    # Undoes our source changes from above.
+    find "${stashed_dir}/${tests_dir_prefix}tests/unit" -type f -name '*.swift' -exec sed -i '' -E "s/import _Motion(.+)/import Motion\1/" {} + || true
+    find "${stashed_dir}/${tests_dir_prefix}src" -type f -name '*.h' -exec sed -i '' -E "s/import \"MotionInterchange.h\"/import <MotionInterchange\/MotionInterchange.h>/" {} + || true
+  }
+  trap reset_imports EXIT
+}
+
+if [ ! -d .kokoro-ios-runner ]; then
+  git clone https://github.com/material-foundation/kokoro-ios-runner.git .kokoro-ios-runner
```

Comment:
Do you want to exit early if this clone fails?

Reply:
Yep, early exiting is handled by `set -e`.

Reply:
Ack.